conf/dbs.config

```diff
@@ -29,21 +29,22 @@ params {
     mgnify_path            = "${params.alphafold2_db}/mgnify/*"
     pdb70_path             = "${params.alphafold2_db}/pdb70/**"
     pdb_mmcif_path         = "${params.alphafold2_db}/pdb_mmcif/*"
-    uniclust30_path        = "${params.alphafold2_db}/uniclust30/*"
+    uniref30_alphafold2_path = "${params.alphafold2_db}/uniref30/**"
```

Comment:
I think that this could be simplified as in #89 but better confirm with @athbaltzis 
```suggestion
    uniref30_alphafold2_path = "${params.alphafold2_db}/uniref30/**"
```

Reply:
what's the difference between the two?🤔 you mean with just 1 *? if so, I agree!!

Reply:
The two asterisks are redundant in this case. I agree let's go with one *

Reply:
I did the changes, cannot commit this one because it's identical!

Reply:
yeah, sorry my fault! 😢